packages/firestore/src/model/document_key.ts

```diff
@@ -51,6 +51,16 @@ export class DocumentKey {
     );
   }
 
+  /** Returns the collection group (i.e. the name of the parent collection) for this key. */
+  getCollectionGroup(): string {
+    return this.path.get(this.path.length - 2);
```

Comment:
Do you want to add this assert? https://github.com/firebase/firebase-js-sdk/pull/5999/files#diff-f464226b90e1b114ea246866fc74fcfcdd4d1b758ad1e38e06fd23970967979aR47

This provides a better error message for `DocumentKey.empty()`

Reply:
Done